packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -1241,6 +1242,19 @@ export function unflagPost(postId: string) {
     };
 }
 
+export function addPostReminder(userId: string, postId: string, timestamp: number) {
```

Comment:
If you use ActionFunc here you don't have to specify types for dispatchfunc etc

Reply:
hang on scratch that, the PR that adds this improvement isnt merged that sorry :P